app/components/home/Home.js

```diff
@@ -128,11 +133,21 @@ export default () => (
       <div className="section">
         <div className="section-content">
           <div className="img-wrapper">
-            <img src={require('../../../src/images/work.jpg')} id="work"></img>
+            <img
+              src={require('../../../src/images/sunsetjump.png')}
```

Comment:
I really like this until it gets to mobile. Not sure what we'd want to replace it with then